### PR TITLE
This pull request fixes :p

### LINKED ISSUE #27849: AI has private channel but can't hear it
(WEB REPORT BY: saulmyers REMOTE: 206.221.180.138:7777)
# Revision
f2ff257a12d56756871ee885022e04bcd6887dc2

# Description
AI has private channel, like here https://i.imgur.com/cdtZUe9.png
But can't hear it or talk on it. You have to use intercomm with speaker "ON" to hear it.
# Steps to Reproduce
Be AI (eugh, who would even do that), try to speak on private channel: nothing. Click "Notes", tweak intercomm to AI private channel frequency. You can't hear what people say near it, UNLESS you enable speaker on nearby intercomms
# What you Expected
To hear my very own private AI channel.
# What Actually Happened
My internal AI headset broked, and I can't hear my special AI channel.

### PR DESCRIPTION
**Pull request by toomykins**
this fixes #27849
AI Private works via subspace systems once more.
Problem was with the Frequency randomization update and its oversight in presets.
AI frequency used to be 1447. which wasn't named per se, but was included in a for loop of included frequencies, hence why it was missed, since AI freq can be any frequency now, those random subspace-included frequencies from 1441 to 1489 often didn't make the AI frequency included. 
AI freq has always worked on the Common server, now it is more obvious.

[bugfix] [oversight]
:cl:
 * bugfix: Fixed private AI channel frequency oversight

### CODE REVIEW

**Comment by PrimeDSS13** [comment]
:p

**Comment by boy2mantwicethefam** [comment]
Fixes what?

**Comment by toomykins** [comment]
:p

File: code/game/machinery/telecomms/presets.dm [hunk 1]
```diff
@@ -66,7 +66,7 @@
 
 /obj/machinery/telecomms/receiver/preset_right/initialize()
 	..()
-	freq_listening = list(RESPONSE_FREQ, COMM_FREQ, ENG_FREQ, SEC_FREQ, COMMON_FREQ) //ert, command, engineering, security
+	freq_listening = list(RESPONSE_FREQ, COMM_FREQ, ENG_FREQ, SEC_FREQ, COMMON_FREQ, AIPRIV_FREQ) //ert, command, engineering, security
```
Comment (DamianX) [inline code/game/machinery/telecomms/presets.dm]: I like how the useless comment is now also outdated

Comment (toomykins) [reply code/game/machinery/telecomms/presets.dm]: why read comments when you can read code, amirite

**Comment by SonixApache** [comment]
:p

**Comment by ShiftyRail** [comment]
:-D

**Comment by toomykins** [comment]
hold up, just realized literally every AI private intercomm is fucking broken too.
gonna have to go through every map.

**Comment by SonixApache** [comment]
i think i reported this a while ago

**Comment by toomykins** [comment]
it has been broken a while

**Comment by toomykins** [comment]
I DONT CARE IT COMPILES AND WORKS SO IT COUNTS

**Comment by toomykins** [comment]
synergy is fucking broken or something all i did was edit a few lines with notepad

**Comment by toomykins** [comment]
giving up

**Pull request closed without merging**